spec/integration/integration_spec_helper.rb

```diff
@@ -64,6 +65,48 @@ def normalize_inspect_output(output)
     gsub!(/(\r\033\[K.*?\r\033\[K).*\r\033\[K(.*)/, "\\1\\2") # strip all progress messages but two
 end
 
+def machinery_boxes
+  {
+    "openSUSE_13_2" => "machinery_132",
+    "openSUSE_13_1" => "machinery_131",
+    "Tumbleweed" => "machinery_tumbleweed"
+  }
+end
+
+def boxes
+  {
+    "openSUSE_13_2" => "opensuse132",
+    "openSUSE_13_1" => "opensuse131"
+  }
+end
+
+def test_according_to_matrix(currently_running_on)
+  path = File.join(Machinery::ROOT, "..", "alfred", "documents", "source", "integration_tests.yml")
```

Comment:
We probably should split up the test matrix in an external and an internal document, so we don't have to cross-reference repositories.